### ISSUE TITLE
Can't connect to nexus

### ISSUE THREAD

**Issue by SometimesRain** [SometimesRain/crazyclient#2]
Some players still can't connect to nexus.

I need someone who's experiencing this bug to compile the client and run it with debugger.

**Comment by michael-elbert** [comment]
I'm willing to compile and debug the issue and I am experiencing the issue, could you give me some details on how to do it? I know how to debug, but what do I need to compile? I have JPEXS but I think that's a decompiler.

**Comment by SometimesRain** [comment]
@michael-elbert  
First set the source up on FlashDevelop (See: http://www.mpgh.net/forum/showthread.php?t=1241694)
Use version 13 debug projector ([Get it here](https://fpdownload.macromedia.com/pub/flashplayer/updaters/13/flashplayer_13_sa_debug.exe))

Press the blue play button to compile and run.
It will open the debug projector and connect it to the IDE.

Make the error happen. The projector should crash but do not close it!
The IDE will open the class that was failing and show an orange circle marking the line that failed.
Console log will probably show some null error. Try to find the variable that was null.

I just need the variable name but you can try to make it work by adding an if statement checking if the value is null before it's used. Null values should not be passed to any of the lower level functions.

**Comment by wyler0** [comment]
Hey Jani,
I'm currently on my Mac and experiencing this issue with your client and hope to help you fix it.

I attempted to install FlashDevelop, but it is not easily installed on Mac. 
I have attempted using Flash Builder 4.7 but this comes with its own slew of errors regarding flashx not being found, bitmap asset not being found, and bunch of others. 
I would try this on my PC but I have no problems with your client there.

If there is another Mac friendly method to try let me know and ill give it a shot.

Edit:
This probably isn't helpful but I ran the client in the projector debugger and received an error about being unable to disable right-click. I cannot reproduce this error, however.
Also, connecting to the proxy server yields the classic server IO error so the problem likely occurs after the initial connection is opened.
